ui/src/actions/beamlineActions.js

```diff
@@ -36,14 +35,18 @@ export function setArgumentValue(cmdName, argIndex, value) {
 }
 
 export function startBeamlineAction(cmdName, parameters, showOutput = true) {
-  return (dispatch) => {
+  return (dispatch, getState) => {
     dispatch(setActionState(cmdName, RUNNING));
 
     if (showOutput) {
       dispatch(showActionOutput(cmdName));
     }
+    const state = getState();
 
-    sendRunBeamlineAction(cmdName, parameters);
+    sendExecuteCommand('beamlineaction', 'beamline_actions', 'run_action', {
```

Comment:
Calling the generic method that runs a "command" on adapter in this case "run_action"